upstream/gitlab_test.go

```diff
@@ -0,0 +1,79 @@
+package upstream
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/h2non/gock"
+	"github.com/simon04/aur-out-of-date/pkg"
+)
+
+func mockGitLab() *gock.Response {
+	return gock.New("https://gitlab.com").
+		Get("/api/v4/projects/gitlab-org%2Fgitlab-ce/repository/tags").
```

Comment:
Replacing `%2F` with `/` (unintuitively) fixes the test. It seems that Gock does not correctly handle escape sequences in the URL path?